tools/rpm-bild.bash

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+echo "installing dependencies"
+
+dnf install -y https://download.copr.fedorainfracloud.org/results/patrickl/libcurl-gnutls/fedora-40-x86_64/07356121-libcurl-gnutls/libcurl-gnutls-8.6.0-8.fc40.x86_64.rpm
+dnf install -y libcurl-devel bison flex gcc make pkg-config
+dnf install -y openssl-devel
```

Comment:
Hmmm ... mixing curl with gnutls and openssl support seems a bit strange. Also this explicit URL confuses me.

Reply:
yeah, the URL is for a dependency that it's not available in dnf, so you need to install it from the official channel.

Why that dependency? because it was the only one recognized by the ./config file.